app/views/episodes/index.html.erb

```diff
@@ -1,9 +1,9 @@
 <% content_for :title, t(".title") %>
 
-<div class="container my-4 mx-2">
-  <div class="d-grid d-sm-flex align-items-center w-auto">
+<div class="my-4">
+  <div class="d-grid d-sm-flex align-items-center w-auto mb-2">
```

Comment:
the `container` class needed to go for all the dashboards. It was redundant with a previous container.